_posts/2017-05-30-database-encryption.markdown

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Database Encryption"
+date: 2017-05-30 11:03
+comments: true
+author: ashkan
+categories: [Database, Encryption, Security]
+---
+
+After examining the data stored in one of our systems, we realized it can potentially include some sensitive user data. To reduce the number of people that are technically able to access the data and reduce the risks associated with a potential data theft, we decided to encrypt data getting persisted in the database.
+
+# Our Goal
+Encrypt database without any downtime.
+
+<!-- more -->
+
+# Approach
+While we want to encrypt existing rows, new rows are constantly being added to our database. In order to reach our goal without downtime there was a clear path:
```

Comment:
Either past or present tense. I think it should all be past.

_While we wanted to encrypt existing rows, new rows were constantly being added. In order to achieve zero downtown we have devised the following path._